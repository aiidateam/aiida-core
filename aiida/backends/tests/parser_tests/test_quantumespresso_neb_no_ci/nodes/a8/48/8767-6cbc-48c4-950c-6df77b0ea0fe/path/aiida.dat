      0.0000000000      0.0000000000      0.0189681992
      0.1851922506      0.0187046241      0.1838448797
      0.3564402704      0.2014600923      0.0420693739
      0.4999981102      0.3979613848      0.0000120856
      0.6435532952      0.2014723687      0.0421794062
      0.8147994577      0.0187077508      0.1839538694
      1.0000000000     -0.0000000254      0.0192781355
