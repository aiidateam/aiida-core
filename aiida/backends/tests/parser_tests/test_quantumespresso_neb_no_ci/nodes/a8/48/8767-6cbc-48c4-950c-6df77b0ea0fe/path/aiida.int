      0.0000000000      0.0000000000
      0.0040000000     -0.0000078310
      0.0080000000     -0.0000298619
      0.0120000000     -0.0000638997
      0.0160000000     -0.0001077512
      0.0200000000     -0.0001592234
      0.0240000000     -0.0002161231
      0.0280000000     -0.0002762572
      0.0320000000     -0.0003374328
      0.0360000000     -0.0003974566
      0.0400000000     -0.0004541355
      0.0440000000     -0.0005052766
      0.0480000000     -0.0005486866
      0.0520000000     -0.0005821725
      0.0560000000     -0.0006035412
      0.0600000000     -0.0006105996
      0.0640000000     -0.0006011545
      0.0680000000     -0.0005730130
      0.0720000000     -0.0005239819
      0.0760000000     -0.0004518681
      0.0800000000     -0.0003544785
      0.0840000000     -0.0002296201
      0.0880000000     -0.0000750996
      0.0920000000      0.0001112759
      0.0960000000      0.0003316995
      0.1000000000      0.0005883644
      0.1040000000      0.0008834637
      0.1080000000      0.0012191905
      0.1120000000      0.0015977377
      0.1160000000      0.0020212987
      0.1200000000      0.0024920663
      0.1240000000      0.0030122338
      0.1280000000      0.0035839942
      0.1320000000      0.0042095407
      0.1360000000      0.0048910662
      0.1400000000      0.0056307640
      0.1440000000      0.0064308271
      0.1480000000      0.0072934486
      0.1520000000      0.0082208216
      0.1560000000      0.0092151392
      0.1600000000      0.0102785944
      0.1640000000      0.0114133805
      0.1680000000      0.0126216904
      0.1720000000      0.0139057174
      0.1760000000      0.0152676543
      0.1800000000      0.0167096945
      0.1840000000      0.0182340309
      0.1880000000      0.0198432863
      0.1920000000      0.0215411271
      0.1960000000      0.0233303075
      0.2000000000      0.0252135113
      0.2040000000      0.0271934224
      0.2080000000      0.0292727246
      0.2120000000      0.0314541017
      0.2160000000      0.0337402376
      0.2200000000      0.0361338161
      0.2240000000      0.0386375212
      0.2280000000      0.0412540366
      0.2320000000      0.0439860461
      0.2360000000      0.0468362337
      0.2400000000      0.0498072832
      0.2440000000      0.0529018785
      0.2480000000      0.0561227033
      0.2520000000      0.0594724415
      0.2560000000      0.0629537770
      0.2600000000      0.0665693936
      0.2640000000      0.0703219752
      0.2680000000      0.0742142057
      0.2720000000      0.0782487688
      0.2760000000      0.0824283484
      0.2800000000      0.0867556283
      0.2840000000      0.0912332925
      0.2880000000      0.0958640248
      0.2920000000      0.1006505089
      0.2960000000      0.1055954288
      0.3000000000      0.1107014684
      0.3040000000      0.1159713113
      0.3080000000      0.1214076416
      0.3120000000      0.1270131430
      0.3160000000      0.1327904994
      0.3200000000      0.1387423947
      0.3240000000      0.1448715126
      0.3280000000      0.1511805371
      0.3320000000      0.1576721520
      0.3360000000      0.1643490411
      0.3400000000      0.1712138883
      0.3440000000      0.1782693774
      0.3480000000      0.1855181923
      0.3520000000      0.1929630169
      0.3560000000      0.2006065349
      0.3600000000      0.2083892327
      0.3640000000      0.2162094465
      0.3680000000      0.2240514264
      0.3720000000      0.2319003390
      0.3760000000      0.2397413504
      0.3800000000      0.2475596271
      0.3840000000      0.2553403354
      0.3880000000      0.2630686416
      0.3920000000      0.2707297122
      0.3960000000      0.2783087135
      0.4000000000      0.2857908118
      0.4040000000      0.2931611735
      0.4080000000      0.3004049649
      0.4120000000      0.3075073524
      0.4160000000      0.3144535023
      0.4200000000      0.3212285810
      0.4240000000      0.3278177549
      0.4280000000      0.3342061902
      0.4320000000      0.3403790534
      0.4360000000      0.3463215108
      0.4400000000      0.3520187288
      0.4440000000      0.3574558737
      0.4480000000      0.3626181118
      0.4520000000      0.3674906095
      0.4560000000      0.3720585332
      0.4600000000      0.3763070493
      0.4640000000      0.3802213240
      0.4680000000      0.3837865237
      0.4720000000      0.3869878148
      0.4760000000      0.3898103636
      0.4800000000      0.3922393366
      0.4840000000      0.3942598999
      0.4880000000      0.3958572201
      0.4920000000      0.3970164634
      0.4960000000      0.3977227962
      0.5000000000      0.3979613848
      0.5040000000      0.3977223602
      0.5080000000      0.3970156305
      0.5120000000      0.3958560281
      0.5160000000      0.3942583855
      0.5200000000      0.3922375348
      0.5240000000      0.3898083086
      0.5280000000      0.3869855392
      0.5320000000      0.3837840589
      0.5360000000      0.3802187002
      0.5400000000      0.3763042953
      0.5440000000      0.3720556768
      0.5480000000      0.3674876768
      0.5520000000      0.3626151278
      0.5560000000      0.3574528622
      0.5600000000      0.3520157123
      0.5640000000      0.3463185105
      0.5680000000      0.3403760892
      0.5720000000      0.3342032807
      0.5760000000      0.3278149174
      0.5800000000      0.3212258317
      0.5840000000      0.3144508559
      0.5880000000      0.3075048224
      0.5920000000      0.3004025636
      0.5960000000      0.2931589119
      0.6000000000      0.2857886995
      0.6040000000      0.2783067589
      0.6080000000      0.2707279225
      0.6120000000      0.2630670225
      0.6160000000      0.2553388915
      0.6200000000      0.2475583617
      0.6240000000      0.2397402655
      0.6280000000      0.2318994353
      0.6320000000      0.2240507034
      0.6360000000      0.2162089023
      0.6400000000      0.2083888642
      0.6440000000      0.2006063646
      0.6480000000      0.1929632423
      0.6520000000      0.1855187847
      0.6560000000      0.1782703088
      0.6600000000      0.1712151313
      0.6640000000      0.1643505691
      0.6680000000      0.1576739392
      0.6720000000      0.1511825583
      0.6760000000      0.1448737434
      0.6800000000      0.1387448113
      0.6840000000      0.1327930789
      0.6880000000      0.1270158632
      0.6920000000      0.1214104808
      0.6960000000      0.1159742488
      0.7000000000      0.1107044839
      0.7040000000      0.1055985032
      0.7080000000      0.1006536233
      0.7120000000      0.0958671613
      0.7160000000      0.0912364339
      0.7200000000      0.0867587581
      0.7240000000      0.0824314507
      0.7280000000      0.0782518286
      0.7320000000      0.0742172087
      0.7360000000      0.0703249078
      0.7400000000      0.0665722428
      0.7440000000      0.0629565306
      0.7480000000      0.0594750881
      0.7520000000      0.0561252321
      0.7560000000      0.0529042795
      0.7600000000      0.0498095471
      0.7640000000      0.0468383519
      0.7680000000      0.0439880108
      0.7720000000      0.0412558405
      0.7760000000      0.0386391579
      0.7800000000      0.0361352800
      0.7840000000      0.0337415236
      0.7880000000      0.0314552056
      0.7920000000      0.0292736428
      0.7960000000      0.0271941522
      0.8000000000      0.0252140505
      0.8040000000      0.0233306547
      0.8080000000      0.0215412816
      0.8120000000      0.0198432482
      0.8160000000      0.0182338007
      0.8200000000      0.0167092829
      0.8240000000      0.0152670778
      0.8280000000      0.0139049919
      0.8320000000      0.0126208315
      0.8360000000      0.0114124029
      0.8400000000      0.0102775124
      0.8440000000      0.0092139665
      0.8480000000      0.0082195713
      0.8520000000      0.0072921334
      0.8560000000      0.0064294589
      0.8600000000      0.0056293543
      0.8640000000      0.0048896259
      0.8680000000      0.0042080801
      0.8720000000      0.0035825231
      0.8760000000      0.0030107614
      0.8800000000      0.0024906012
      0.8840000000      0.0020198489
      0.8880000000      0.0015963109
      0.8920000000      0.0012177935
      0.8960000000      0.0008821030
      0.9000000000      0.0005870457
      0.9040000000      0.0003304281
      0.9080000000      0.0001100564
      0.9120000000     -0.0000762630
      0.9160000000     -0.0002307237
      0.9200000000     -0.0003555195
      0.9240000000     -0.0004528440
      0.9280000000     -0.0005248908
      0.9320000000     -0.0005738536
      0.9360000000     -0.0006019261
      0.9400000000     -0.0006113019
      0.9440000000     -0.0006041746
      0.9480000000     -0.0005827381
      0.9520000000     -0.0005491858
      0.9560000000     -0.0005057114
      0.9600000000     -0.0004545087
      0.9640000000     -0.0003977713
      0.9680000000     -0.0003376928
      0.9720000000     -0.0002764668
      0.9760000000     -0.0002162872
      0.9800000000     -0.0001593474
      0.9840000000     -0.0001078412
      0.9880000000     -0.0000639623
      0.9920000000     -0.0000299042
      0.9960000000     -0.0000078607
      1.0000000000     -0.0000000254
