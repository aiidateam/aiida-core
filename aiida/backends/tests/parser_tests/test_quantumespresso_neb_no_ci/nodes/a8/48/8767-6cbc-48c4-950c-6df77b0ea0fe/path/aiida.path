RESTART INFORMATION
       4
      20
       0
NUMBER OF IMAGES
   7
ENERGIES, POSITIONS AND GRADIENTS
Image:    1
     -143.1194029084
      0.000000000000      0.000000000000      0.000000000000      0.000000000000      0.000000000000      0.000000000000  0  0  0
      3.779452265771      3.779452265771      3.667061212199      0.000000000000      0.000000000000      0.000350106405  0  0  1
      3.779452265771      3.779452265771      0.346777655993      0.000000000000      0.000000000000     -0.000368872775  0  0  1
      3.779452265771      0.000000000000      4.188234623531      0.000000000000      0.000000000000      0.000148540580  0  0  1
      0.000000000000      3.779452265771      4.188234623531      0.000000000000      0.000000000000      0.000148540580  0  0  1
Image:    2
     -143.1187155261
      0.000000000000      0.000000000000      0.000000000000      0.000000000000      0.000000000000      0.000000000000
      3.779452265771      3.779452265771      3.568944928889      0.000000000000      0.000000000000      0.007054019771
      3.779452265771      3.779452265771      0.218264485442      0.000000000000      0.000000000000     -0.004896654315
      3.779452265771      0.000000000000      4.028373592367      0.000000000000      0.000000000000     -0.004483225205
      0.000000000000      3.779452265771      4.028373592367      0.000000000000      0.000000000000     -0.004483225205
Image:    3
     -143.1119993860
      0.000000000000      0.000000000000      0.000000000000      0.000000000000      0.000000000000      0.000000000000
      3.779452265771      3.779452265771      3.650807131711      0.000000000000      0.000000000000      0.038882322584
      3.779452265771      3.779452265771      0.102199413211      0.000000000000      0.000000000000     -0.022046596689
      3.779452265771      0.000000000000      3.876905370931      0.000000000000      0.000000000000     -0.011458295833
      0.000000000000      3.779452265771      3.876905370931      0.000000000000      0.000000000000     -0.011458295833
Image:    4
     -143.1047780960
      0.000000000000      0.000000000000      0.000000000000      0.000000000000      0.000000000000      0.000000000000
      3.779452265771      3.779452265771      3.826693694438      0.000000000000      0.000000000000      0.000000000000
      3.779452265771      3.779452265771      0.000001937424      0.000000000000      0.000000000000      0.000000000000
      3.779452265771      0.000000000000      3.826696375990      0.000000000000      0.000000000000      0.000000000000
      0.000000000000      3.779452265771      3.826696375990      0.000000000000      0.000000000000      0.000000000000
Image:    5
     -143.1119989348
      0.000000000000      0.000000000000      0.000000000000      0.000000000000      0.000000000000      0.000000000000
      3.779452265771      3.779452265771      4.002579639263      0.000000000000      0.000000000000     -0.038882985159
      3.779452265771      3.779452265771     -0.102190038366      0.000000000000      0.000000000000      0.022043249700
      3.779452265771      0.000000000000      3.776489250795      0.000000000000      0.000000000000      0.011459356909
      0.000000000000      3.779452265771      3.776489250795      0.000000000000      0.000000000000      0.011459356909
Image:    6
     -143.1187154112
      0.000000000000      0.000000000000      0.000000000000      0.000000000000      0.000000000000      0.000000000000
      3.779452265771      3.779452265771      4.084445514692      0.000000000000      0.000000000000     -0.007056659156
      3.779452265771      3.779452265771     -0.218253499750      0.000000000000      0.000000000000      0.004899124557
      3.779452265771      0.000000000000      3.625023769525      0.000000000000      0.000000000000      0.004483426904
      0.000000000000      3.779452265771      3.625023769525      0.000000000000      0.000000000000      0.004483426904
Image:    7
     -143.1194029094
      0.000000000000      0.000000000000      0.000000000000      0.000000000000      0.000000000000      0.000000000000
      3.779452265771      3.779452265771      3.986329625988      0.000000000000      0.000000000000     -0.000357535893
      3.779452265771      3.779452265771     -0.346777655993      0.000000000000      0.000000000000      0.000374900078
      3.779452265771      0.000000000000      3.465156214656      0.000000000000      0.000000000000     -0.000147998559
      0.000000000000      3.779452265771      3.465156214656      0.000000000000      0.000000000000     -0.000147998559
