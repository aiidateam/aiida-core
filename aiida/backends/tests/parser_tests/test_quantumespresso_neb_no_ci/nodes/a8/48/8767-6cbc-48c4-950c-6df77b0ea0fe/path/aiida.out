
     Program NEB v.5.2.0 (svn rev. 11612) starts on 27Jul2015 at 16:27: 6 

     This program is part of the open-source Quantum ESPRESSO suite
     for quantum simulation of materials; please cite
         "P. Giannozzi et al., J. Phys.:Condens. Matter 21 395502 (2009);
          URL http://www.quantum-espresso.org", 
     in publications or presentations arising from this work. More details at
     http://www.quantum-espresso.org/quote

     Parallel version (MPI), running on     6 processors
     R & G space division:  proc/nbgrp/npool/nimage =       6
     Reading input from pw_1.in
     Reading input from pw_2.in

     initial path length           =  1.2762 bohr
     initial inter-image distance  =  0.2127 bohr
 
     string_method                 =    neb
     restart_mode                  =    from_scratch
     opt_scheme                    =    broyden
     num_of_images                 =    7
     nstep_path                    =    20
     CI_scheme                     =    no-CI
     first_last_opt                =    F
     use_freezing                  =    F
     ds                            =    2.0000 a.u.
     k_max                         =    0.3000 a.u.
     k_min                         =    0.2000 a.u.
     suggested k_max               =    0.1542 a.u.
     suggested k_min               =    0.1028 a.u.
     path_thr                      =    0.2000 eV / A

     ------------------------------ iteration   1 ------------------------------

     tcpu =      0.2    self-consistency for image   1
     tcpu =      4.1    self-consistency for image   2
     tcpu =      8.7    self-consistency for image   3
     tcpu =     12.8    self-consistency for image   4
     tcpu =     17.1    self-consistency for image   5
     tcpu =     21.6    self-consistency for image   6
     tcpu =     25.8    self-consistency for image   7

     activation energy (->) =   0.397961 eV
     activation energy (<-) =   0.397961 eV

     image        energy (eV)        error (eV/A)        frozen

         1      -3894.4770104            0.018968            T
         2      -3894.3805774            1.435133            F
         3      -3894.1879765            1.567149            F
         4      -3894.0790490            0.000000            F
         5      -3894.1879766            1.566910            F
         6      -3894.3805773            1.435389            F
         7      -3894.4770105            0.019278            T

     path length          =  1.276 bohr
     inter-image distance =  0.213 bohr

     ------------------------------ iteration   2 ------------------------------

     tcpu =     30.0    self-consistency for image   2
     tcpu =     35.0    self-consistency for image   3
     tcpu =     39.8    self-consistency for image   4
     tcpu =     41.2    self-consistency for image   5
     tcpu =     46.0    self-consistency for image   6

     activation energy (->) =   0.397961 eV
     activation energy (<-) =   0.397961 eV

     image        energy (eV)        error (eV/A)        frozen

         1      -3894.4770104            0.018968            T
         2      -3894.4477683            0.587638            F
         3      -3894.3040596            0.491243            F
         4      -3894.0790490            0.000052            F
         5      -3894.3040406            0.491045            F
         6      -3894.4477726            0.587212            F
         7      -3894.4770105            0.019278            T

     path length          =  1.438 bohr
     inter-image distance =  0.240 bohr

     ------------------------------ iteration   3 ------------------------------

     tcpu =     51.2    self-consistency for image   2
     tcpu =     55.7    self-consistency for image   3
     tcpu =     59.7    self-consistency for image   4
     tcpu =     61.4    self-consistency for image   5
     tcpu =     65.5    self-consistency for image   6

     activation energy (->) =   0.397961 eV
     activation energy (<-) =   0.397961 eV

     image        energy (eV)        error (eV/A)        frozen

         1      -3894.4770104            0.018968            T
         2      -3894.4592425            0.374346            F
         3      -3894.2583261            0.211604            F
         4      -3894.0790490            0.000045            F
         5      -3894.2583197            0.211324            F
         6      -3894.4592454            0.374346            F
         7      -3894.4770105            0.019278            T

     path length          =  1.515 bohr
     inter-image distance =  0.253 bohr

     ------------------------------ iteration   4 ------------------------------

     tcpu =     70.4    self-consistency for image   2
     tcpu =     74.7    self-consistency for image   3
     tcpu =     78.9    self-consistency for image   4
     tcpu =     80.8    self-consistency for image   5
     tcpu =     84.9    self-consistency for image   6

     activation energy (->) =   0.397961 eV
     activation energy (<-) =   0.397961 eV

     image        energy (eV)        error (eV/A)        frozen

         1      -3894.4770104            0.018968            T
         2      -3894.4583058            0.183845            F
         3      -3894.2755503            0.042069            F
         4      -3894.0790490            0.000012            F
         5      -3894.2755381            0.042179            F
         6      -3894.4583027            0.183954            F
         7      -3894.4770105            0.019278            T

     path length          =  1.501 bohr
     inter-image distance =  0.250 bohr

     ---------------------------------------------------------------------------


     neb: convergence achieved in   4 iterations
 
     NEB          :  1m27.36s CPU     1m29.20s WALL

 
   This run was terminated on:  16:28:35  27Jul2015            

=------------------------------------------------------------------------------=
   JOB DONE.
=------------------------------------------------------------------------------=
